app/routes/tasks.js

```diff
@@ -24,4 +25,9 @@ export default class TasksRoute extends Route {
       );
     }
   };
+
+  @action
+  loading() {
+    return true;
+  }
```

Comment:
When are we setting this to false?

Reply:
https://guides.emberjs.com/release/routing/loading-and-error-substates/#toc_the-loading-event
by doing this ember searches for `tasks-loading.hbs`.

Reply:
Well, they're setting it false after the promise resolves. Are we doing that?

Reply:
Well, I found out we don't even have to do that. Just add `routename-loading` file.

Reply:
but if we have 15 routes tomorrow, we'll need 15 `*-loading.hbs* files, right? 🤔